admin_ui/src/views/Login.vue

```diff
@@ -3,77 +3,67 @@
         <div class="inner">
             <h1>{{ siteName }}</h1>
             <form v-on:submit.prevent="login">
-                <label>Username</label>
-                <input
-                    name="username"
-                    type="text"
-                    v-model="username"
-                />
+                <label>{{ $t("Username") }}</label>
+                <input name="username" type="text" v-model="username" />
 
-                <label>Password</label>
-                <input
-                    name="password"
-                    type="password"
-                    v-model="password"
-                />
+                <label>{{ $t("Password") }}</label>
+                <input name="password" type="password" v-model="password" />
 
-                <button>Login</button>
+                <button>{{ $t("Login") }}</button>
             </form>
         </div>
     </div>
 </template>
 
-
-<script>
+<script lang="ts">
 import axios from "axios"
 
 export default {
     data: function () {
         return {
             username: "",
-            password: "",
+            password: ""
         }
     },
     computed: {
         siteName() {
             return this.$store.state.metaModule.siteName
         },
+        defaultLanguage() {
+            return this.$store.state.metaModule.defaultLanguage
+        }
```

Comment:
I think we can remove this as well since it's not being used.

Reply:
You're right - thanks. Have removed it now.

Reply:
Great. Can you please add `__init__.py` to the translation module (otherwise we get `ModuleNotFoundError`). The docs should also be updated with the new features and it would be great if you could add a new image as the drop down now looks a lot better.